frontend/src/components/Switch.tsx

```diff
@@ -26,9 +27,9 @@ const SwitchLabel = styled.label<{
   width: ${(props) => (props.width ? props.width + 'px' : '100px')};
   height: ${(props) => (props.width ? props.width * 0.5 + 'px' : '50px')};
   background: lightgray;
-  background: ${(props) => (props.isOn ? (props.onColor ? props.onColor : '#468751') : 'lightgray')};
+  background: ${(props) => (props.isOn ? (props.onColor ? props.onColor : '#468751') : '#F5F5F5')};
```

Comment:
why are there 2 backgrounds?

Reply:
props.onColor is to set another color if want another color, otherwise is the normal green color